crates/wash-lib/src/cli/label.rs

```diff
@@ -0,0 +1,79 @@
+use anyhow::{bail, Result};
+use clap::Parser;
+
+use crate::{
+    common::{boxed_err_to_anyhow, find_host_id},
+    config::WashConnectionOptions,
+};
+
+use super::{CliConnectionOpts, CommandOutput};
+
+#[derive(Debug, Clone, Parser)]
+pub struct LabelHostCommand {
+    #[clap(flatten)]
+    pub opts: CliConnectionOpts,
+
+    /// ID of host to update the actor on. If a non-ID is provided, the host will be selected based
+    /// on matching the prefix of the ID or the friendly name and will return an error if more than
+    /// one host matches.
+    #[clap(name = "host-id")]
+    pub host_id: String,
+
+    /// Delete the label, instead of adding it
+    #[clap(long = "delete", default_value = "false")]
+    pub delete: bool,
+
+    /// Host label in the form of a `[key]=[value]` pair, e.g. "cloud=aws". When `--delete` is set, only the key is provided
```

Comment:
Note: Let's create an issue to add support for multiple labels since I am almost certain people will ask for that

Reply:
https://github.com/wasmCloud/wasmCloud/issues/1286